src/stories/containers/Finances/components/CardNavigationFinance/CardNavigationFinance.tsx

```diff
@@ -0,0 +1,58 @@
+import styled from '@emotion/styled';
+import lightTheme from '@ses/styles/theme/light';
+import React from 'react';
+import CardNavigationGeneric from '../CardNavigationGeneric';
+import ReadMore from '../ReadMore';
+
+interface Props {
+  svgImage: JSX.Element;
+  title: string;
+  description: string | JSX.Element;
```

Comment:
you can modify the type here to `description: React.ReactNode` as `ReactNode` extends from `string` too

Reply:
done